lib/eigensolve_quda.cpp

```diff
@@ -455,7 +455,7 @@ namespace quda
       if (kSpace[0]->Location() == QUDA_CPU_FIELD_LOCATION) {
         kSpace[0]->Source(QUDA_RANDOM_SOURCE);
       } else {
-        RNG *rng = new RNG(kSpace[0]->Volume(), 1234, kSpace[0]->X());
+        RNG *rng = new RNG(*kSpace[0], 1234);
```

Comment:
nothing new, but 1234 here seems weird. Should that be singled out in a constexpr with a name. Just for code maintainability.

Reply:
Is there any way we can pass a global seed to QUDA via `--seed 1234`?

Reply:
In a subsequent pull we could clean up the seed setting to be supplied at command line like @cpviolator suggests.  If we do that, the question is what to do when we have multiple RNGs active?  E.g., in multigrid we have different RNGs active for different grids.

Reply:
Anyway, out of scope for this pull, so marking this as resolved.